docs/manual/external-tools.tex

```diff
@@ -391,63 +391,50 @@
 
 \sectionAndLabel{IntelliJ IDEA}{intellij}
 
-% The following method has been tested with IntelliJ IDEA 15.0.3.
+% The following method has been tested with IntelliJ IDEA 2019.1.1.
 
 This section tells you how to make IntelliJ IDEA automatically run a
-checker on every compile.  If you want to run the checker from a terminal
-within IntelliJ IDEA, follow the command-line instructions in
-Section~\ref{javac-installation}.
+checker on every compile for Java projects and/or modules.  If your project
```

Comment:
Is this "if" telling the user something they should do instead, or something they should do in addition?
Does it enable type-checking on every compile?

Reply:
These instructions do not work if the project uses a build tool.  I updated the text.